src-tauri/src/swarm.rs

```diff
@@ -144,9 +157,20 @@ pub fn get_swarm_processes() -> String {
         return "".to_string();
     }
 
+    let app_data_dir = tauri::api::path::home_dir().expect("🙈 Failed to get app data directory");
+    let app_data_dir = app_data_dir
+        .join(".config/prem");
+    let app_data_dir = app_data_dir
+        .to_str()
+        .expect("🙈 Failed to convert app data dir path to str");
+
+    let prem_app_env = format!("{app_data_dir}/miniconda/envs/prem_app");
```

Comment:
could pass in `handle` then:

```suggestion
    let env = create_environment(handle);
    let python_path = PathBuf::from(env.get(&String::from("PREM_PYTHON")).unwrap().to_string());
    let prem_app_env = python_path.parent().unwrap().parent().unwrap().to_str().unwrap();
```

Reply:
do we want to launch the `create_env.sh` script every time we want to get the processes? It seems too much to me  🤔 (now the `get_swarm_processes` function is run every 500ms)

Reply:
why every 500ms? Can qe do 10x less?